apps/meteor/app/lib/server/functions/setUserActiveStatus.ts

```diff
@@ -115,6 +115,9 @@ export async function setUserActiveStatus(userId: string, active: boolean, confi
 	if (!active && !settings.get('Accounts_Send_Email_When_Deactivating')) {
 		return true;
 	}
+	if (!user.emails) {
```

Comment:
should this be a short circuit for all other empty possibilities as well? like:

```suggestion
	if (!user.emails || !Array.isArray(user.emails) || user.emails.length === 0) {
```

if so, the `Array.isArray` can be removed from line 123